pcss-client/Clients/JudicialCalendarsServicesClient.cs

```diff
@@ -26,164 +26,20 @@ private Newtonsoft.Json.JsonSerializerSettings CreateSerializerSettings()
 
         public async Task<ICollection<JudicialCalendar>> JudicialCalendarsGetAsync(string locationId, DateTime startDate, DateTime endDate, System.Threading.CancellationToken cancellationToken)
         {
-            //         [HttpGet("api/calendar/judges")]
-            // public HttpResponseMessage ReadCalendars([FromUri] int[] locationIds = null, string startDate = null, string endDate = null)
+            var requestUrl = $"https://wsgw.test.jag.gov.bc.ca/courts/catsAPI/api/v2/calendar/judges?locationIds={locationId}&startDate={startDate:dd-MMM-yyyy}&endDate={endDate:dd-MMM-yyyy}";
+            var request = new HttpRequestMessage(HttpMethod.Get, requestUrl);
+            request.Headers.Add("Authorization", "Basic " + Convert.ToBase64String(System.Text.Encoding.ASCII.GetBytes("CATS_API:4R,;UTGFm6ff")));
```

Comment:
@TimaQT was wondering if we can change this so that it pulls the credentials as an environment variable? Even making it blank would work for now just so the credentials is not visible.

Reply:
will update them, wanted to do it too